### ISSUE TITLE
Report version term code sometimes reports multiple times

### ISSUE THREAD

**Issue by RedBeard0531** [mintty/mintty#606]
```
user@host ~
$ echo -ne '\e[>c'

user@host ~
$ 77;20602;0c77;20602;0c
```

For some reason it isn't exactly consistent. Sometimes it only outputs once, other times it repeats itself 3 times, but twice is by far the most common. I started looking into this because vim uses that code for its `t_RV` setting which I think is why it is using `ttymouse=xterm` rather than `ttymouse=sgr`.

**Comment by mintty** [comment]
I don't know why it's inconsistent; sometimes even a screenful of responses are generated.
But the reason it happens at all is that the output of the response sequence (beginning like the request) causes another request. This can be fixed.

**Comment by mintty** [comment]
Fixed in 2.7.0.
